docs/content/users/install/phpstorm.md

```diff
@@ -22,6 +22,17 @@ It’s easiest to use the DDEV Integration Plugin, which you can install from [i
 
 Install and enable the plugin, then [set up `phpunit`](#enabling-phpunit) since it doesn’t yet handle that for you.
 
+### Additional Step for Mutagen
+
+Please note that if you use mutagen (If you use Colima you will use mutagen by default.  If you use Docker Desktop, you may opt-into using Mutagen), you will need an additional configuration task to setup your PHP Interpreter even if you are using this plugin.  
+
+1. Open Preferences
+2. Navigate to PHP 
```

Comment:
Removing this trailing space should fix the linting error [Randy pointed out](https://github.com/ddev/ddev/pull/4732#issuecomment-1463967247).

```suggestion
2. Navigate to PHP
```